packages/terra-clinical-result/tests/jest/result-name-header-cell/ResultNameHeaderCell.test.jsx

```diff
@@ -13,6 +13,16 @@ describe('ResultNameHeaderCell', () => {
     expect(cell).toMatchSnapshot();
   });
 
+  it('should render correctly when a full result name is provided', () => {
+    const cell = shallow(<ResultNameHeaderCell resultName="Name" fullResultName="Full result name" />);
+    expect(cell).toMatchSnapshot();
```

Comment:
we should also explicitly check to see if the name is properly set
```suggestion
    expect(cell).toMatchSnapshot();
```

Reply:
Good catch. Fixed: https://github.com/cerner/terra-clinical/pull/867/commits/361d8279f67328213f80517a2e1ad39a1400c5cf#diff-7d79d57b26e94be506526eda6c255971437300a335ab6c307c2f721a2a12b6f4R18